### ISSUE TITLE
name of KSLLIW's intrinsic function

### ISSUE THREAD

**Issue by linsinan1995** [riscv/riscv-p-spec#41]
Hi, I am Sinan from PLCT doing p extension implementation on gnu toolchain.
 
The intrinsic function of KSLLIW has a quite unique name in the spec, `rvksllw` instead of `__rv__ksllw`. I wonder is it a typo or not.

**Comment by chuanhua** [comment]
Yes. It is a typo (missing the leading space to make the line as code) which causes an ASCIIDOC formatting error. Thanks for finding this issue. Will be fixed in the next release.

**Comment by pz9115** [comment]
Out of date, mark as closed.
